vertx-openjdk-apache-mysql-on-ubuntu/readme.md

```diff
@@ -0,0 +1,4 @@
+# Vert.x, OpenJDK, Apache, and MySQL Server on Ubuntu VM
+
+This template uses the Azure Linux CustomScript extension to deploy Vert.x, OpenJDK, Apache, and MySQL Server on Ubuntu 14.04 LTS.
```

Comment:
Probably a good idea to explain a bit more about Vert.x and the scenario this template accomplishes and what it doesn't do